sos/report/plugins/openvswitch.py

```diff
@@ -79,17 +79,19 @@ def setup(self):
             # Capture DPDK and other parameters
             "ovs-vsctl -t 5 get Open_vSwitch . other_config",
             # Capture OVS list
-            "ovs-vsctl list Open_vSwitch",
+            "ovs-vsctl -t 5 list Open_vSwitch",
             # Capture OVS interface list
-            "ovs-vsctl list interface",
+            "ovs-vsctl -t 5 list interface",
```

Comment:
"ovs-vsctl list bridge" below and further down also need "-t 5"

Reply:
will do